source/features/close-out-of-view-modals.tsx

```diff
@@ -6,7 +6,7 @@ const observer = new IntersectionObserver(([{intersectionRatio, target}]) => {
 	if (intersectionRatio === 0) {
 		observer.unobserve(target);
 		const dropdown = select(`
-			.dropdown-details[open] summary,
+			details[open] summary,
```

Comment:
This would also select `details` in comments.

I think the new class is `details-overlay`

```suggestion
			.details-overlay[open] summary,
```

Reply:
Done.